config.json

```diff
@@ -199,6 +199,17 @@
         "mathematics"
       ]
     },
+    {
+      "slug": "armstrong-numbers",
+      "uuid": "a639ce2f-48e3-46cd-9d11-955ba2a9eaaa",
+      "core": false,
+      "unlocked_by": null,
```

Comment:
With v2 right in front of us we need it to be either ``core`` or ``unlocked_by`` something.

I'm not very keen on touching core again before the release, but I am also not sure what core exercise should unlock this exercise, as I am not comfortable with the exercise itself.

Reply:
Hm, the best "company" for that exercise would probably be the other maths-heavy exercises like sum of multiples, largest series product, difference of squares. But each of those is unlocked by a different core exercise.

Reply:
Whather it can be seen as a maths-heavy exercise depends on the approach, though. It can be done by integer and modulo division as in my reference solution, but also by converting the number to a string and working with that, as I expect most people to do it...

Reply:
Yes, thats mostly because for an initial get-going the first ~10 exercises were used as ``core``, while the remaining were distributed more or less evenly or by difficulty among those cores to get unlocked.

My plan was to re-address this after the release together with the mentors, when we better know what students are struggling with.

But if it's really a 2, then I think it can be safely unlocked by ``leap`` or ``collatz``, both are mainly dealing with numbers and their characteristics as well.

Reply:
I was thinking `collatz` as well. As for the difficulty rating, I mainly went with the estimation in other tracks which put it at 2.